share/rocm/cmake/ROCMInstallTargets.cmake

```diff
@@ -157,6 +159,24 @@ function(rocm_install_targets)
                         NAMELINK_ONLY
             )
         endif()
+        if(T_TYPE MATCHES ".*_LIBRARY" AND ROCM_SYMLINK_LIBS AND NOT ROCM_CREATED_${PROJECT_NAME}_SYMLINK)
+            set(ROCM_CREATED_${PROJECT_NAME}_SYMLINK TRUE CACHE INTERNAL "Record symlink created.")
+            string(TOLOWER "${PROJECT_NAME}" LINK_SUBDIR)
+            if(CMAKE_HOST_WIN32)
+                set(SYMLINK_CMD "file(COPY \${LINK_TGT} DESTINATION \${LINK_NAME})")
+            else()
+                set(SYMLINK_CMD "execute_process(COMMAND ln -sf ../lib \${LINK_NAME})")
```

Comment:
Can test later but this LINK_NAME is just lib for rocblas right?   So you are in the project subfolder already.

Reply:
No, it's a bit more complex due to needing to create the symlink in the package, this method of doing it creates this symlink in the correct location at install time, which may be a more complex location (e.g. when CPack is creating the packages, the link is created directly inside CPack's internal staging directory).